### PR TITLE
Changes to subnets Getting Started and Overview guides

### PR DESCRIPTION
**Pull request by mefrem**
### Description

Updates to subnets documentation, specifically the Getting Started guide and Overview page, the result of feedback.

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -5,33 +5,27 @@ title: Getting Started
...
 # Getting Started
 
-Developers can test their applications on a subnet either locally or on Hiro's
-hosted testnet subnet. This page describes two different walkthroughs that
-illustrate how to use a subnet.
+You understand subnets from the [overview](https://docs.hiro.so/subnets/overview)—now you can test one out in action. Hiro's Clarinet can serve a local subnet with [`clarinet integrate`](https://docs.hiro.so/clarinet/how-to-guides/how-to-run-integration-environment) as one of the networks in your Stacks development environment.
 
-- Run a local subnet
-- Use Hiro's subnet on testnet
+This guide walks a user through a **subnet demonstration**—minting and transferring NFTs between a main chain (local devnet) and a subnet—to showcase subnet's high throughput and low latency functionality. By the end of this guide, the user will
```
Comment (jbencin) [inline docs/getting-started.md:10]: > subnet-to

I'm not sure this is correct use of a hyphen here (and in a couple places too). Consider a period, colon, or ellipsis instead

**Review by jbencin — COMMENTED** [review]
Took a quick look and added a few comments. Also one general comment: Should we be consistently capitalizing all our product names like Subnets and Stacks.js?

**Comment by mefrem** [comment]
Thanks Jeff, incorporated all your suggestions (and capitalized all instances of Stacks.js... I'm not sure why, but I think subnets has always been uncapitalized. Not sure why or who to confirm that with.)

I believe we're good to merge.

File: docs/overview.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ This diagram outlines the interaction between a subnet and the Stacks layer-1 ch
 
 ## Features
 
-A subnets is designed to transact on Stacks assets, meaning users can move assets from the Stacks chain in and out of a subnet. While a user's assets are in a subnet, the asset is locked in the subnet contract on the Stacks chain.
+A subnet is designed to on and off-board Stacks assets, meaning users can move assets from the Stacks chain in and out of a subnet. While a user's assets are in a subnet, the asset is locked in the subnet contract on the Stacks chain, and representations of those assets are created—appearing in a user's Hiro Wallet—and handled by applications on the subnet. The benefit of these subnet applications is the higher throughput and faster block time relative to Stacks mainnet.
```
Comment (jbencin) [inline docs/overview.md:29]:
```suggestion
A subnet is designed to temporarily hold Stacks assets. Users can deposit assets from the Stacks chain, take advantage of faster transactions and lower fees while on the subnet, and withdraw them when finished.
While a user's assets are in a subnet, the asset is locked in the subnet contract on the Stacks chain, and representations of those assets are created—appearing in a user's Hiro Wallet—and handled by applications on the subnet.
```

**Review by jbencin — COMMENTED** [review]
Looks good, made a couple more suggestions though that I missed the first time. I'll give @obycode an opportunity to review this as well, but if he's busy I can approve.

Also here's a general Markdown tip: Write one sentence per line. It will be rendered the same, but it makes `git diff`s easier to read

**Comment by mefrem** [comment]
> Also here's a general Markdown tip: Write one sentence per line. It will be rendered the same, but it makes git diffs easier to read

cc: @LakshmiLavanyaKasturi — thanks Jeff!

**Review by jbencin — APPROVED** [review]

**Pull request merged**